src/cloud/client.rs

```diff
@@ -5,12 +5,14 @@ use std::io;
 use std::path::PathBuf;
 use std::time::Duration;
 
+use anyhow::Context;
 use surf::http::auth::{AuthenticationScheme, Authorization};
 
 use crate::options::CloudOptions;
 use crate::platform::config_dir;
 
 const EDGEDB_CLOUD_BASE_URL: &str = "https://free-tier0.ovh-us-west-2.edgedb.cloud";
+const EDGEDB_CLOUD_DEFAULT_DNS_ZONE: &str = "ovh-us-west-2.edgedb.cloud";
```

Comment:
These are obsolete.  The default API URL should be `https://api.g.<dns-zone>` and there is no `dns-zone` default, it's always determined from the token.

Reply:
So the user should provide a `dns-zone` in `edgedb cloud login`?

Reply:
Ah, good point.  Let's use `aws.edgedb.cloud` as the default for logins for now.